content/Certification/certifying_a_product/development.md

```diff
@@ -0,0 +1,200 @@
++++
+title = "Development"
+chapter = false
+weight = 6
++++
+
+![](../imgs/development.png)
+
+The process of developing a Matter product is highly dependent on the device
+being developed, the platform being used and the policies of the company. There
+is no single "best" way to integrate Matter into a product. This guide aims to
+address the common considerations for Matter product development.
+
+SDK development guides are available to aid in the process of software
+development and system and data model design. Information on Matter
+implementations is available at The SDK documentation is available in the
+[Matter Handbook SDK documentation](https://handbook.buildwithmatter.com/sdk/).
+
+Information on the CSA open source SDK is available at the
+[Matter SDK documentation site](https://project-chip.github.io/connectedhomeip-doc/index.html).
+
+Integration of the SDK on specific platforms is covered by the
+[Platform guides](https://project-chip.github.io/connectedhomeip-doc/platforms/index.html).
+
+
+## Matter-specific product considerations
+The documentation in this section aims to give an brief overview of some of the
+common product and factory considerations and challenges that may differ from
+non-Matter products.
+
+Matter device require some material to be provisioned at the factory, after
+certification. Some of these materials need to be provisioned on a per-device
+basis, some are per-product-line.
+
+![](../img/factory_data.png)
+
+Most platforms provide a comprehensive factory data solution customized to
+Matter materials. Please see the documentation for your selected
+[Platform](https://project-chip.github.io/connectedhomeip-doc/platforms/index.html).
+
+### Device Attestation Certificates and Certification Authorities
+
+Device attestation certificates are used to attest devices as being authentic
```

Comment:
Probably could use a note here about a lot of the procedures being applicable to certification; as a result:
* developers early in the process may use test VID, test PID, and test CAs
* developers should become members early in the process to understand the details of the security procedures within the Alliance

Reply:
I'm not sure i totally get what you mean here, but nonetheless I took a stab at it. Added a subsection for development DACs at the bottom. PTAL. 
![image](https://github.com/user-attachments/assets/1ac1f976-6623-4677-9e81-d39f9d8b57cd)